src/ray/raylet/local_object_manager.cc

```diff
@@ -85,26 +85,67 @@ void LocalObjectManager::FlushFreeObjectsIfNeeded(int64_t now_ms) {
   }
 }
 
+int64_t LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_required) {
+  if (RayConfig::instance().object_spilling_config().empty() ||
+      !RayConfig::instance().automatic_object_spilling_enabled()) {
+    return num_bytes_required;
+  }
+
+  RAY_LOG(INFO) << "Choosing objects to spill of total size " << num_bytes_required;
```

Comment:
Can we make it DEBUG? It is probably redundant because we have other INFO message right after this.

Reply:
Hmm I'd prefer to keep since they actually print different variables.